flink-external-resources/flink-external-resource-gpu/src/test/java/org/apache/flink/externalresource/gpu/GPUDriverTest.java

```diff
@@ -78,7 +81,12 @@ public void testGPUDriverWithTestScriptDoNotExist() throws Exception {
 	@Test(expected = FlinkException.class)
 	public void testGPUDriverWithInexecutableScript() throws Exception {
 		final Configuration config = new Configuration();
-		config.setString(GPUDriver.DISCOVERY_SCRIPT_PATH, "src/test/resources/inexecutable-gpu-discovery.sh");
+		final TemporaryFolder temporaryFolder = new TemporaryFolder(new File("."));
```

Comment:
I would try to avoid forcing crate the temporary folder at current path if possible.
We can omit the constructor argument, and set `executableFile.getAbsolutePath()` to the configuration.